wp1-frontend/src/components/CreateSimpleList.vue

```diff
@@ -11,8 +56,34 @@
 import SecondaryNav from './SecondaryNav.vue';
 export default {
   components: { SecondaryNav },
-  name: 'CreateSimpleList'
+  name: 'MyLists',
+  data: function() {
+    return {
+      wikiProjects: ['en.wikipedia.org']
+    };
+  },
+  created: function() {
+    this.getWikiProjects();
+  },
+  methods: {
+    getWikiProjects: async function() {
+      const response = await fetch(`${process.env.VUE_APP_API_URL}/sites/`);
+      var data = await response.json();
+      this.wikiProjects = data.sites;
+      this.wikiProjects.forEach((element, index) => {
+        if (element === 'https://en.wikipedia.org') {
+          this.wikiProjects.splice(index, 1);
+          return;
+        }
```

Comment:
I know in Python you get bugs if you modify a list while you're iterating over it. The same is probably true in Javascript? Why not just:

```suggestion
      index = this.wikiProjects.indexOf('https://en.wikipedia.org')
      this.wikiprojects.splice(index, 1);
```